functions.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+
+def readData(filePath): # we will probably use os to do the file import later, but this works for now
+    pathExt = filePath.split('.')
+    ext = pathExt[1]
+    
+    if ext == 'csv':
+        data = pd.read_csv(filePath)
+    elif (ext == 'xlsx') or (ext == 'xls') or (ext == 'xlsm') or (ext == 'xlsb'):
+        #sheet = input("Please enter the sheet name or number (first sheet = 0, second = 1, etc.)")
+        data.pd.read_excel(filePath)
+    return(data)
+
+
+def get_yn():
```

Comment:
is this a helper for a command line utility? if so, please give me an example of how to test your code out in the PR's description